src/pages/ProfilePage.tsx

```diff
@@ -266,38 +247,31 @@ function ProfilePage(props) {
                         icon={Illustrations.ToddBehindCloud}
                         iconWidth={variables.modalTopIconWidth}
                         iconHeight={variables.modalTopIconHeight}
-                        title={props.translate('notFound.notHere')}
+                        title={translate('notFound.notHere')}
                         shouldShowLink
-                        link={props.translate('notFound.goBackHome')}
                     />
                 )}
             </View>
         </ScreenWrapper>
     );
 }
 
-ProfilePage.propTypes = propTypes;
-ProfilePage.defaultProps = defaultProps;
 ProfilePage.displayName = 'ProfilePage';
 
-export default compose(
-    withLocalize,
-    withOnyx({
-        personalDetails: {
-            key: ONYXKEYS.PERSONAL_DETAILS_LIST,
-        },
-        session: {
-            key: ONYXKEYS.SESSION,
-        },
-        report: {
-            key: ({route, session}) => {
-                const accountID = Number(lodashGet(route.params, 'accountID', 0));
-                const reportID = lodashGet(ReportUtils.getChatByParticipants([accountID]), 'reportID', '');
-                if ((session && Number(session.accountID) === accountID) || Session.isAnonymousUser() || !reportID) {
-                    return null;
-                }
-                return `${ONYXKEYS.COLLECTION.REPORT}${reportID}`;
-            },
+export default withOnyx<ProfilePageProps, ProfilePageOnyxProps>({
+    personalDetails: {
+        key: ONYXKEYS.PERSONAL_DETAILS_LIST,
+    },
+    report: {
+        key: ({route, session}) => {
+            const accountID = Number(route.params?.accountID ?? 0);
+            const reportID = ReportUtils.getChatByParticipants([accountID])?.reportID ?? '';
+
+            if ((session && Number(session.accountID) === accountID) ?? SessionActions.isAnonymousUser() ?? !reportID) {
+                return `${ONYXKEYS.COLLECTION.REPORT}}`;
```

Comment:
Still considering about better approach

Reply:
Maybe it makes sense to ask author about clarification on this part, but anyway returning the whole collection doesn't look right.

Reply:
It is not the whole collection - just a string without exact report id  - `'report_' ` (same result in logic as it was with null )

Reply:
'report_'  is a collection key, I guess it returns not a collection cause you have extra `}` in your case

Reply:
Before it was a null - results both have the empty value (nonexistent collection) 
We do not have `report_` collection  in the storage